akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -1824,7 +1825,7 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
         if (totChunks == statusTotChunks)
           statusCount += 1
         else {
-          statusCount = ThreadLocalRandom.current.nextInt(0, totChunks)
+          statusCount = ThreadLocalRandom.current.nextInt(0, totChunks).toLong
```

Comment:
that is a warning I don't understand the purpose of. When is it unsafe to assign an Int to a Long field?

Reply:
I agree it is rather loud - I guess it is intended to catch code where you might 'accidentally' use a Long where an Int would have been sufficient?

Seems reasonable to disable these warnings, I'll give that a try